_1327/documents/forms.py

```diff
@@ -55,6 +57,11 @@ def __init__(self, *args, **kwargs):
 			self.fields['group'].widget = forms.HiddenInput()
 			self.fields['group'].required = False
 
+		# remove temp prefix from url title in initial data if there is one
+		if self.initial['url_title'].startswith('temp_'):
```

Comment:
hypothetically, if a user would name the url title `temp_fail` - that would fail, right?